cloud/aws/nlb/modules.tf

```diff
@@ -0,0 +1,7 @@
+module "filter-tags" {
+  source = "github.com/claranet/terraform-signalfx-detectors.git//common/filter-tags"
+
+  filter_defaults        = "filter('env', '${var.environment}') and filter('sfx_monitored', 'true')"
```

Comment:
```suggestion
  filter_defaults        = "filter('aws_tag_env', '${var.environment}') and filter('sfx_monitored', 'true')"
```

Reply:
actually should the sfx_monitored be aws_tag_sfx_monitored too?